include/radio_types.h

```diff
@@ -1010,7 +1020,7 @@ G_STATIC_ASSERT(sizeof(RadioCellInfoNr) == 112);
 typedef struct radio_cell_info_1_4 {
     guint32 registered RADIO_ALIGNED(1);
     guint32 connectionStatus RADIO_ALIGNED(4);
-    guint8 cellInfoType RADIO_ALIGNED(1);
+    RADIO_CELL_INFO_TYPE_1_4 cellInfoType RADIO_ALIGNED(1);
```

Comment:
Meh, let's keep it a `guint8` since it's arbitrary aligned, and to be closer to the hidl-generated code. Can take the enum though, it you need it.